### PR TITLE
Dread: fix hide scans with nothing data

### LINKED ISSUE #6658: Option to hide item scans doesn't work in Dread
image
I was under the impression that the option to hide an item's scan translates to the name that appears on the map when you look at the item. 
image
It seems odd that we would have support for hiding the item name when picking up the item, but not when looking at the map, given that the opposite way around seems like the most natural configuration to use when playing with this option.
image

### PR DESCRIPTION
**Pull request by Miepee**
Fixes #6658

### CODE REVIEW

File: test/games/test_log_file_export.py [hunk 1]
```diff
@@ -66,7 +66,7 @@ def test_layout_patch_data_export(
 
     data = factory.create_data()
 
-    # _update_committed(data_path, cosmetic_path, data, cosmetic_patches)
+    _update_committed(data_path, cosmetic_path, data, cosmetic_patches)
```
Comment (dyceron) [inline test/games/test_log_file_export.py:69]: Nice

Comment (Miepee) [reply test/games/test_log_file_export.py:69]: ops

File: test/games/test_log_file_export.py [hunk 2]
```diff
@@ -109,6 +109,7 @@ def pytest_generate_tests(metafunc: _pytest.python.Metafunc) -> None:
         "dread/custom_start.rdvgame",  # crazy settings
         "dread/custom_patcher_data.rdvgame",  # custom patcher data
         "dread/all_settings.rdvgame",  # all settings enabled
+        "dread/hide_all_with_nothing.rdvgame",  # Model+scan+name hidden with nothing data
```
Comment (dyceron) [inline test/games/test_log_file_export.py:112]: This test seems a but unnecessary the only difference is two lines are slightly different. The rest of the rdvgame is standard.

Comment (Miepee) [reply test/games/test_log_file_export.py:112]: this test is to ensure that nothing breaks this in the future.
Its the easiest way to test things, ideas welcome for another test.

Comment (Miepee) [reply test/games/test_log_file_export.py:112]: also unsure where you got your two lines from?
every time `"map_icon": {`is used, its different now

**Review by henriquegemignani — APPROVED** [review]
YOUR NOTHING

**Pull request merged**